xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
Actually, it is the other way around.  The first signal **"on"** the diverging route (advancedAspect) is another double head signal that is showing a diverging aspect.  Since the only choice was Clear, ourAspect was being set to Clear (green/red) even though the route was diverging.

The indication in the aspects.xml file for Diverging Clear: _Proceed through diverging route_.


Dave Sand